src/client/datascience/jupyter/jupyterRequest.ts

```diff
@@ -15,7 +15,7 @@ export function createAuthorizingRequest(getAuthHeader: () => any) {
             const authorizationHeader = getAuthHeader();
             const keys = Object.keys(authorizationHeader);
             keys.forEach((k) => origHeaders.append(k, authorizationHeader[k].toString()));
-            origHeaders.append('Content-Type', 'application/json');
+            origHeaders.set('Content-Type', 'application/json');
```

Comment:
Append means there might already be a content-type set. We want to force application/json. Synapse was saying bad request when there were two content-types.